manifests/l/Logitech/Solar/1.20.28/Logitech.Solar.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Logitech.Solar
+PackageVersion: 1.20.28
+PackageLocale: en-US
+Publisher: Logitech Inc.
```

```suggestion
Publisher: Logitech
```
The Publisher just shows up as Logitech for me in ARP